tensorflow/compiler/xla/service/pattern_matcher.h

```diff
@@ -2157,6 +2221,19 @@ XLA_NULLOP_PATTERN(PartitionId)
 XLA_NULLOP_PATTERN(ReplicaId)
 #undef XLA_NULLOP_PATTERN
 
+// Matches if the instruction is unary and has any of the given opcodes or is
+// the operand.
+template <typename Arg>
```

Comment:
Please give this a more descriptive name than `arg`.

Reply:
Are you referring to `arg` or `Arg`? Do you have a suggestion for an alternative name?

Reply:
both arg and Arg.

`arg` is...what, a `pattern`?  `base_pattern`?  Either of those sgtm I think.